templates/openstackconfiggenerator/config/16.2/nic/net-config-multi-nic.role.j2.yaml

```diff
@@ -138,6 +140,11 @@ resources:
                   list_concat_unique:
                     - get_param: {{ $net.Name }}Routes
                     - get_param: {{ $net.Name }}InterfaceRoutes
+{{`{%- if '`}}{{ $net.Name }}{{`' in role.default_route_networks %}`}}
```

Comment:
should we also have/keep the condition from [1] in L51 for the {{ $net.Name }}InterfaceDefaultRoute ?

[1] https://github.com/openstack/tripleo-heat-templates/blob/stable/train/network/config/multiple-nics/role.role.j2.yaml#L62

Reply:
Doesn't really matter since the param has a default value